aspnet/CircularGauge/Legend.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
 title: Legend | CircularGauge | ASP.NET Webforms | Syncfusion
-description:  legend
+description:  Learn here about legends in CircularGauge control for Syncfusion ASP.NET WebForm controls its elements and more.
```

Comment:
Can you change description like as below,
 "Learn here about legend support in Syncfusion ASP.NET Web Forms Circular Gauge control, along with its various elements and additional features."

Reply:
done